src/deploymentmanager.ts

```diff
@@ -551,7 +552,40 @@ export class DeploymentManager implements IDeploymentManager {
         data.push('PCS_TELEMETRY_STORAGE_TYPE=' + outputs.telemetryStorageType.value);
         data.push('PCS_TSI_FQDN=' + outputs.tsiDataAccessFQDN.value);
 
-        console.log('Copy the following environment variables to /scripts/local/.env file: \n\ %s', `${chalk.cyan(data.join('\n'))}`);
+        this.setEnvironmentVariables(data);
+
+        console.log('Please save the following environment variables to /scripts/local/.env file: \n\ %s', `${chalk.cyan(data.join('\n'))}`);
+    }
+
+    private setEnvironmentVariables(data: string[]) {
+        data.forEach((envvar) => {
+            let cmd = '';
+            switch ( os.type() ) {
+                case 'Windows_NT': {
+                    cmd = 'SETX ' + envvar;
+                    break;
+                }
+                case 'Darwin': {
+                    cmd = 'launchctl setenv ' + envvar;
+                    break;
+                }
+                case 'Linux': {
+                    const space = /\s/;
+                    envvar = envvar.replace(space, '=');
+                    cmd = 'echo ' + envvar + ' >> /etc/environment';
+                    break;
+                }
+                default: { 
+                    console.log('The environment could not be set. unable to determine OS.');
+                    break; 
+                 }
+            }
+            cp.exec(cmd);
+
+        });
+
+        //fs.writeFileSync(environmentFileName, data.join('\n'));
+        //console.log('The environment variables have been set on this machine. A copy of these variables is saved in the file  \n\ %s', '.env');
```

Comment:
remove?

Reply:
Just keeping here, if we wish to add this functionality later.

Reply:
I would delete the code that is commented.